api/lecture_halls.go

```diff
@@ -368,6 +368,8 @@ func createLectureHall(c *gin.Context) {
 		CombIP: req.CombIP,
 		PresIP: req.PresIP,
 		CamIP:  req.CamIP,
+		CameraIP: req.CameraIP,
```

Comment:
`CamIP` vs. `CameraIP` is a bit confusing, maybe we should rename it in a later PR. It is fine for now though.